contents/docs/error-tracking/_snippets/hosting-source-maps.mdx

```diff
@@ -0,0 +1,58 @@
+If you don't upload source maps, PostHog will try to fetch them from your minified JavaScript bundles. This means your source maps must be publicly accessible without authentication.
+
+We **recommend** uploading your source maps to PostHog instead for better reliability.
+
+### Local development
+
+Since source maps need to be publicly accessible for PostHog to fetch them, they won't be available when you're testing your app on `localhost`. You can upload your source maps to PostHog using the `posthog-cli` or you can use a proxy service like [ngrok](https://ngrok.com/) to make your local server accessible publicly.
```

Comment:
```suggestion
Since source maps need to be publicly accessible for PostHog to fetch them, they won't be available when you're testing your app on `localhost` unless you build your app. Once built, you can upload your source maps to PostHog using the `posthog-cli` or you can use a proxy service like [ngrok](https://ngrok.com/) to make your local server accessible publicly.
```